content/project/sparta/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "SPARTA"
+date: 2020-03-11T14:31:37-04:00
+logo: "/images/research/sparta.png"
+tags: ["cybersecurity"]
+homepage: "http://www.sparta.eu"
+facebook: ""
+linkedin: ""
+twitter: ""
+youtube: ""
+funding_bodies: ["horizon2020"]
+eclipse_projects: ["technology.steady"]
+project_type: running
+summary: "Re-imagining the way cybersecurity research, innovation, and training are performed in Europe"
+---
+Cybersecurity is an urgent and major societal challenge. Highly correlated with the digitalization of our societies, cyberthreats have an increasing impact on our lives. It is therefore essential to ensure digital security and strategic autonomy of the EU by strengthening leading cybersecurity capacities. This challenge will require the coordination of Europe’s best competences, towards common research and innovation goals. 
+
+SPARTA is a novel Cybersecurity Competence Network, supported by the EU’s H2020 program, with the objective to develop and implement top-tier research and innovation collaborative actions. Strongly guided by concrete challenges forming an ambitious Cybersecurity Research & Innovation Roadmap, SPARTA will setup unique collaboration means, leading the way in building transformative capabilities and forming a world-leading Cybersecurity Competence Network across the EU. From basic human needs (health) to economic activities (energy, finance, and transport) to technologies (ICT and industry) to sovereignty (eGovernment, public administration), four research and innovation programs will push the boundaries to deliver advanced solutions to cover emerging challenges. 
+
+The SPARTA consortium assembles, for the first time, 44 actors from more than 14 Member States, bringing together the expertise of different kind of stakeholders, be they research institutes, private organizations or national authorities,
+Re-imagining the way cybersecurity research, innovation, and training are performed in Europe across domains and expertise, from foundations to applications, in academia and industry. SPARTA will create a long-lasting community capable of collaboration to define, develop, share, and evolve solutions that will help practitioners prevent cybercrime and enhance cybersecurity. 
+In sharing experiences and excellence, challenges and capabilities, SPARTA makes decisive contributions to European strategic autonomy.
+
```

Comment:
Please see the ['Markdown Here' cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables) for how to create tables without HTML.

Reply:
Thx for the links, I will your cheatsheet for our tables.

Reply:
Someone made a change to include unsafe code in the rendering engine, which is what is allowing that through. I'll be removing that soon as it shouldn't be set

Reply:
> Please see the ['Markdown Here' cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables) for how to create tables without HTML.

I was not so successful to create tables using the suggested syntax, @autumnfound . Actually, I could not find how to do a colspan or how to increase the default width of a column:
![image](https://user-images.githubusercontent.com/6196602/78807794-dad27d00-79c4-11ea-8149-22caacb2df96.png)

Any suggestion? Thx

Reply:
There is no way in GitHub Flavoured Markdown tables (which is the exact spec we are using) to use colspan in Markdown. There would have to be a custom component for that given table to enable colspan for this table in current state. Where would you need colspan for your table?

Reply:
@autumnfound I don't really like the result using markdown syntax for this table.
Why do you recommend not using HTML syntax instead?
Thanks